OpenDreamRuntime/Procs/Native/DreamProcNativeHelpers.cs

```diff
@@ -438,8 +443,15 @@ internal static partial class DreamProcNativeHelpers {
     /// <summary>
     /// Gets the turf 1 step away from an atom in the given direction
     /// </summary>
-    public static DreamObjectTurf? GetStep(AtomManager atomManager, IDreamMapManager mapManager, DreamObjectAtom loc, AtomDirection dir) {
-        var dirInt = (int)dir;
+    public static DreamObjectTurf? GetStep(AtomManager atomManager, IDreamMapManager mapManager, DreamObjectAtom loc,
+        AtomDirection dir) {
+        return GetStep(atomManager, mapManager, loc, (int)dir);
+    }
+
+    /// <summary>
+    /// Gets the turf 1 step away from an atom in the given direction
+    /// </summary>
+    public static DreamObjectTurf? GetStep(AtomManager atomManager, IDreamMapManager mapManager, DreamObjectAtom loc, int dirInt) {
```

Comment:
Probably better to have the calling code cast the dir to an AtomDirection instead of creating a version of this method that takes an int.